pages/migrations/generate.mdx

```diff
@@ -1,5 +1,128 @@
 # Generate migration
 
+## From `0.76.0` and above
+
+In this version, we are removing the `run` function from the `generate` configuration.
+
+Instead, we're introducing a new package called `@snaplet/client`.
```

Comment:
@jgoux I'm going to do the renames for `client` to `seed` and base this branch against https://github.com/snaplet/docs/pull/40 so we have a single PR to merge when we're ready to go live. Hope that makes sense.